test/integration/targets/collections_runtime_pythonpath/ansible-collection-python-dist-boo/setup.cfg

```diff
@@ -0,0 +1,53 @@
+[metadata]
+name = ansible-collections.python.dist
+version = 1.0.0rc2.post3.dev4
+url = https://github.com/your-handle/ansible-collection-python-dist-boo
+project_urls =
+    Bug Tracker = https://github.com/your-handle/ansible-collection-python-dist-boo/issues
+    CI: GitHub = https://github.com/your-handle/ansible-collection-python-dist-boo/actions
+    Source Code = https://github.com/your-handle/ansible-collection-python-dist-boo
+description = Example Ansible Collection containing 'boo' Ansible module
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+author = Sviatoslav Sydorenko (@webknjaz), Ansible Core Team
+author_email = info+github/collection-py-dist-example@ansible.com
+license = GPLv3+
+license_file = LICENSE
+classifiers =
+    Development Status :: 2 - Pre-Alpha
+
+    Environment :: Console
+
+    Intended Audience :: Developers
+    Intended Audience :: Information Technology
+    Intended Audience :: System Administrators
+
+    License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)
+
+    Operating System :: OS Independent
+    Operating System :: POSIX
+
+    Programming Language :: Python
+
+    Topic :: Software Development
+    Topic :: Software Development :: Libraries
+    Topic :: Software Development :: Libraries :: Python Modules
+keywords =
+    Ansible Collections
```

Comment:
This metadata is not required by the test. The test should focus on what is needed to verify collection loading. This looks more like an example of how to publish a collection for PyPI, which is something that should go in an example repo or docs rather than a test.

```suggestion
```

Reply:
I don't like the idea of having a broken dist in tests.

Reply:
How is it broken?